osu.Game/Rulesets/UI/Scrolling/DrawableScrollingRuleset.cs

```diff
@@ -178,6 +163,36 @@ protected override void LoadComplete()
                 throw new ArgumentException($"{nameof(Playfield)} must be a {nameof(ScrollingPlayfield)} when using {nameof(DrawableScrollingRuleset<TObject>)}.");
         }
 
+        private ScrollVisualisationMethod visualisationMethod = ScrollVisualisationMethod.Sequential;
+
+        public ScrollVisualisationMethod VisualisationMethod
+        {
+            get => visualisationMethod;
+            set
+            {
+                visualisationMethod = value;
+                updateScrollAlgorithm();
+            }
+        }
+
+        private void updateScrollAlgorithm()
```

Comment:
Out of scope of this pull, but I majorly like this, since it will help fix issues like SVs not taking effect immediately pretty much instantly (just subscribe to `ControlPoints` updates and call this).